terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -402,7 +402,7 @@ public void test06_Query_Display() {
 
         // output 06_08 Test
         assertThat(driver.findElement(By.xpath(
-                "//a[contains(@href, '?page=1&size=10&age=0&countries=&_dateOfBirth=&main=false&name=TEST%5B%5D%23%2B%3D%26TEST')]"))
+                "//a[contains(@href, '?page=1&size=10&age=0&countries=&_dateOfBirth=&main=&name=TEST%5B%5D%23%2B%3D%26TEST')]"))
                 .getText(), is("2"));
     }
```

Comment:
- [ ] It is better to have 3 patterns of `main=true` / `main=false` / `main=`.
It is especially important that `false` / `null` are bound as intended.

Reply:
added Boolean false pattern to test case 06_03.

3b696f16a7f6d722295abfb194bb9c212216936a